### PR TITLE
docs(profile): add more descriptions

### PR DESCRIPTION
**Pull request by AvidCoder101**
@ming-tsai Wanted to contribute, and noticed that the README was a little empty, so I just added a little more info. Please tell me if I need to change anything or if I have any errors. Thanks (:

### CODE REVIEW

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @AvidCoder101,
Could you check how to made the markdown more beautiful?

**Comment by AvidCoder101** [comment]
Of course, could you give some suggestions on what to do, like if you'd like a banner?

**Comment by ming-tsai** [comment]
> Of course, could you give some suggestions on what to do, like if you'd like a banner?

Ok, I will give a suggestion

**Comment by ming-tsai** [comment]
> Of course, could you give some suggestions on what to do, like if you'd like a banner?

Hi @AvidCoder101,
This is my recommendation.

**Code**
```md
## Hi there 👋, we are Make Contributions
<!--

**Here are some ideas to get you started:**

🙋‍♀️ A short introduction - what is your organization all about?
🌈 Contribution guidelines - how can the community get involved?
👩‍💻 Useful resources - where can the community find your docs? Is there anything else the community should know?
🍿 Fun facts - what does your team eat for breakfast?
🧙 Remember, you can do mighty things with the power of [Markdown](https://guides.github.com/features/mastering-markdown/)
-->

We are an organization that wants to encourage Github newbies to come and participate in open source projects and contribute more to Github (well, the clue in the name 😉).

### There's plenty to take part in the organizing, check out some of our repositories below:

- [DSA](https://github.com/MakeContributions/DSA) - Data Structure and Algorithm (DSA) contributions
- [markdown-dungeon](https://github.com/MakeContributions/markdown-dungeon) - A dungeon made fully out of markdown.
- [ideahub](https://github.com/MakeContributions/ideahub) - A collection of ideas and projects that contain from beginner to advance :octocat: 🎯🚀

> One of our maintainers can add you to the organization once you contribute if you'd like to join, and you could be part of the growing community of over 200 members. Do you want to contact us? Email tsaibot@outlook.com
```

**Rendered**
## Hi there 👋, we are Make Contributions


We are an organization that wants to encourage Github newbies to come and participate in open source projects and contribute more to Github (well, the clue in the name 😉).

### There's plenty to take part in the organizing, check out some of our repositories below:

- [DSA](https://github.com/MakeContributions/DSA) - Data Structure and Algorithm (DSA) contributions
- [markdown-dungeon](https://github.com/MakeContributions/markdown-dungeon) - A dungeon made fully out of markdown.
- [ideahub](https://github.com/MakeContributions/ideahub) - A collection of ideas and projects that contain from beginner to advance :octocat: 🎯🚀

> One of our maintainers can add you to the organization once you contribute if you'd like to join, and you could be part of the growing community of over 200 members. Do you want to contact us? Email tsaibot@outlook.com

**Comment by AvidCoder101** [comment]
Thanks, I will add this is and anything else I think is relevant :)

**Comment by AvidCoder101** [comment]
@ming-tsai I changed it

File: profile/README.md [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
+- [markdown-dungeon](https://github.com/MakeContributions/markdown-dungeon) - A dungeon made fully out of markdown.
+- [ideahub](https://github.com/MakeContributions/ideahub) - A collection of ideas and projects that contain from beginner to advance :octocat: 🎯🚀
+
+> One of our maintainers can add you to the organisation once you contribute if you wish to join, and you could be part of the growing community of over 200 members.
+
+> Thanks for taking a look at our organisation! Wish to contact us? Email tsaibot@outlook.com
+
+
+
+
```
Comment (ming-tsai) [inline profile/README.md]:
```suggestion
## Hi there 👋, we are Make Contributions
<!--

**Here are some ideas to get you started:**

🙋‍♀️ A short introduction - what is your organization all about?
🌈 Contribution guidelines - how can the community get involved?
👩‍💻 Useful resources - where can the community find your docs? Is there anything else the community should know?
🍿 Fun facts - what does your team eat for breakfast?
🧙 Remember, you can do mighty things with the power of [Markdown](https://guides.github.com/features/mastering-markdown/)
-->

We are an organization that wants to encourage Github newbies to come and participate in open source projects and contribute more to Github (well, the clue in the name 😉).

### There's plenty to take part in the organizing, check out some of our repositories below:

- [DSA](https://github.com/MakeContributions/DSA) - Data Structure and Algorithm (DSA) contributions
- [markdown-dungeon](https://github.com/MakeContributions/markdown-dungeon) - A dungeon made fully out of markdown.
- [ideahub](https://github.com/MakeContributions/ideahub) - A collection of ideas and projects that contain from beginner to advance :octocat: 🎯🚀

> One of our maintainers can add you to the organization once you contribute if you'd like to join, and you could be part of the growing community of over 200 members. Do you want to contact us? Email tsaibot@outlook.com

```

**Review by ming-tsai — APPROVED** [review]

**Pull request merged**